src/generated/java/com/prowidesoftware/swift/model/field/SRU2024FieldEnum.java

```diff
@@ -0,0 +1,746 @@
+/*
+* Copyright 2006-2024 Prowide
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.prowidesoftware.swift.model.field;
+
+/**
+* Enumeration of fields for SRU 2024
+*
+* @since 9.5.2
+*/
+public enum SRU2024FieldEnum {
```

Comment:
Remove the SRU from the name so we can have git history on the source and also we avoid breaking compile on SRU changes. We do not need it because the SRU is implicit in the whole library. Notice Field classes do not have the SRU neither in the name or package. The SRU in the javadoc you can keep it.

Reply:
Ok